db/migrate/20200703161150_create_listing_endorsements.rb

```diff
@@ -0,0 +1,11 @@
+class CreateListingEndorsements < ActiveRecord::Migration[6.0]
+  def change
+    create_table :listing_endorsements do |t|
```

Comment:
should this be called `classified_listing_endorsements`, this way we keep the tables grouped?